MOTIVATION.md

```diff
@@ -0,0 +1,138 @@
+# Motivation
+
+The Human Cell Atlas (HCA) Data Coordination Platform (DCP) ingests,
+processes, and stores single-cell data with the goal of making that data easily
+available to a wide audience of researchers.
+The single-cell data type of greatest interest to most researchers is expression
+matrices, matrices that contain cell-by-gene expression values that are the
+starting point for many downstream analyses.
+The importance of this data type calls for a specialized interface that
+maximizes its utility. This specialized interface is called the DCP "matrix
+service". The matrix service is responsible for enabling investigators to access
+and analyze HCA expression data as easily as possible.
+
+# Expression Matrix Interfaces
+
+The native interface exposed by the DCP storage system, along with the quirks of
+the DCP's data organization, would make working with the matrices challenging.
+In contrast, the matrix service aims to incorporate HCA expression data into
+researchers' existing analysis practices seamlessly.
+
+As an illustration, those familiar with dataframe manipulation should be able
+to work with HCA expression data as they might work with their own local data:
+
```

Comment:
_should be able_ also sounds aspirational. Could we globally replace all instances with "will" or "can" or more confident language?

Reply:
:+1: https://www.youtube.com/watch?v=7o1DCJwG2qI